src/provider/view/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+
+import {ClassNameBuilder} from '../../utils/ClassNameBuilder';
+
+import * as Styles from './Icon.module.scss';
+
+export enum IconState {
+    DEFAULT = 'default',
+    DISABLED = 'disabled',
+    ACTIVE = 'active'
```

Comment:
Is `ACTIVE` something that should be a prop? Seems like this is internal state of the component, rather than something that should be set via props?

Reply:
This file was pulled from OFD so a lot of the stuff isn't used here. I agree it should be moved up a level. I'll remove them here and when I move it over to the UI library I'll correct in OFD.